fiftyone/core/dataset.py

```diff
@@ -3692,7 +4230,7 @@ def _save(self, view=None, fields=None):
             self._deleted = True
             raise ValueError("Dataset '%s' is deleted" % name)
 
-    def _save_field(self, field):
+    def _save_field(self, field, _enforce_read_only=True):
```

Comment:
I assume the reason for us to pass this as a private argument is that we don't want to publicly publish it?

Reply:
In general, private methods are already "undocumented, call at your own risk" methods. But when it comes to bypassing things like read-only constraints, just prefer to send an extra signal, even to internal callers, to not use this parameter unless you reallllly know what you're doing